crates/tabby-common/src/api/event.rs

```diff
@@ -114,6 +113,7 @@ impl From<Event> for LogEntry {
     fn from(event: Event) -> Self {
         Self {
             ts: timestamp(),
+            user: None,
```

Comment:
is there other place use this `From<Event>` trait? if not, we can just remove it.